source/Junkership.js

```diff
@@ -5,18 +5,24 @@ import Reference from "./Reference.js"
 import Projectile from "./Projectile.js"
 import Score from "./Score.js"
 
-var junkerTex = Pixi.Texture.fromImage(require("./images/blue-starship.png"))
-
 export default class Junkership extends Pixi.Sprite {
     constructor() {
-        super(junkerTex)
+        super(PIXI.loader.resources.redJunkership.texture)
         game.playerCount++
         this.speed = 60
         this.score = new Score()
+
+        this.hitBox = new Pixi.Rectangle( this.x + 1 , this.y + 1 ,
+            this.width - 3 , this.height - 3 )
```

Comment:
So at this point, why aren't we using `this.anchor` yet? This would make this math much less confusing.